3rdparty/celt-0.11.0-build/celt-0.11.0-build.pro

```diff
@@ -64,13 +64,11 @@ unix {
 
 SOURCES *= bands.c celt.c cwrs.c entcode.c entdec.c entenc.c header.c kiss_fft.c laplace.c mathops.c mdct.c modes.c pitch.c plc.c quant_bands.c rate.c vq.c
 
-CONFIG(debug, debug|release) {
-  CONFIG += console
-  DESTDIR = ../../debug
-}
-
-CONFIG(release, debug|release) {
-  DESTDIR = ../../release
-}
+# TARGET is the same in both versions of CELT, thus we have to specify
+# a separated folder to prevent objects collision between the two.
+OBJECTS_DIR = $$OBJECTS_ROOT/3rdparty/celt-0.11.0-build
```

Comment:
We don't do subdirs anymore, so drop `3rdparty`. Also, drop the `-build` suffix, since we use target names in all other cases but here -- and having "build" in the "target" seems odd.

Reply:
The problem is that `-build` is appended automatically to the build directory name with other 3rdparty projects.

I think it happens because TARGET contains the project file name when `compiler.pri` is included

Reply:
Yes, that's the case.

What do you think about defining `TARGET` before including `compiler.pri`?

Reply:
I am not sure about how putting TARGET before compiler.pri would look.
Let's just go with -build for now, and do the TARGET cleanup later?

Reply:
Sounds good.